api/app/__init__.py

```diff
@@ -43,6 +44,7 @@
     user=os.getenv('SMTP_USER'),
     password=os.getenv('SMTP_PASSWORD'),
 )
+aws_ses_client = AwsSesClient()
```

Comment:
Are we planning to use SES?

Reply:
Probably later, but there's a whole suite of AWS SES tests that we can keep in if we just initialise this client here.